### PR TITLE
Main Library - Ajustes de maquetación

### PR DESCRIPTION
**Pull request by fermarinsanchez**
fix(leebrary): add color to mediaType selector elements (icons) on AssetList header

### CODE REVIEW

File: plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ const AssetList = ({
         value: item.key,
         label: item.name,
         icon: (
-          <Box style={{ height: 16, marginBottom: 5 }}>
+          <Box style={{ height: 16, marginBottom: 5, color: '#636D7D' }}>
```
Comment (johan-fx) [inline plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js:290]: There is no token for this color? maybe you can use the "sx={(theme) => ({ color: theme.some.token })}" props?

**Review by johan-fx — CHANGES_REQUESTED** [review]
There is a way to not hardcode this color?

File: plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js (hunk 1 above)
Comment (fermarinsanchez) [reply plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js]: problem solved. Thanks!

**Review by johan-fx — APPROVED** [review]
looks good to me

**Pull request merged**